manifests/server/install.pp

```diff
@@ -1,5 +1,7 @@
 # == Class dns::server
 #
+include dns::server::default
```

Comment:
You can do this in the puppet 4 / future parser, does this not work in the traditional parser?

Kyle Anderson wrote:
> In manifests/server/install.pp <https://github.com/ajjahn/puppet-dns/pull/130#discussion_r33468120>:
>
>> @@ -1,5 +1,7 @@
>>  # == Class dns::server
>>  #
>> +include dns::server::default
>
> This is not correct, you can't put an include outside of a class definition
>
> —
> Reply to this email directly or view it on GitHub <https://github.com/ajjahn/puppet-dns/pull/130/files#r33468120>.
>

-- 
Johnson Earls
darkfoxprime@gmail.com